store/auth.go

```diff
@@ -97,13 +101,13 @@ func RequestPackageAccessMacaroon() (string, error) {
 }
 
 // DischargeAuthCaveat returns a macaroon with the store auth caveat discharged.
-func DischargeAuthCaveat(username, password, macaroon, otp string) (string, error) {
+func DischargeAuthCaveat(username, password, caveat, otp string) (string, error) {
```

Comment:
Can we please reorder arguments like this: `(caveat, username, password, otp)`

Feels a bit (arguably a tiny bit) more expectable.